kafka/pom.xml

```diff
@@ -97,6 +99,12 @@
             <version>${jackson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Because of the `@Nonnull` and `@Nullable` annotations in the `KafkaTokenStore`. Mostly needed when moving to 4.6.0 but I figured there was little sense in reverting in between.